<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatebox</class>
 <widget class="QWidget" name="updatebox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="bt_update">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="consoleOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_changelog">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>73</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Changelog :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_askupdate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>There is an update available. Do you want to update ?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
